services/storage/signed-uploads/src/upsert.js

```diff
@@ -0,0 +1,46 @@
+import AWS from 'aws-sdk'; // eslint-disable-line import/no-extraneous-dependencies
+
+const bucket = process.env.BUCKET;
+const region = process.env.REGION;
+
+const s3 = new AWS.S3({ signatureVersion: 'v4', region });
+
+/* eslint-disable import/prefer-default-export, no-console */
```

Comment:
1. The global "eslint disable" rules should be placed at the top of the file
2. `no-console` is not necessary